elixir/apps/web/lib/web/components/table_components.ex

```diff
@@ -250,19 +255,26 @@ defmodule Web.TableComponents do
       </.vertical_table_row>
   """
 
+  attr :label_width, :string, default: nil
+  attr :value_width, :string, default: nil
```

Comment:
Maybe it's better to name them like:
```suggestion
  attr :label_class, :string, default: nil
  attr :value_class, :string, default: nil
```

Reply:
Yeah, that's fair.  I was trying to guide the user in to was type of value was meant to be put there, but looking at it now it probably is better to just open it up to letting any class value be set.

Reply:
This has been updated